src/index.ts

```diff
@@ -55,6 +101,11 @@ const createGenerator: CodegenGeneratorConstructor = (config, context) => {
 
 			await emit('apiImpl', path.join(outputPath, relativeSourceOutputPath, 'impl', `${context.generator().toIdentifier(group.name)}.ts`), 
 				{ ...rootContext, ...group, ...doc }, false, hbs)
```

Comment:
If we declare:

```typescript
const apiContainsMultipartOperations = containsMultipartOperations(operations)
```

Then we add property into this object, `{ containsMultipartOperations: apiContainsMultipartOperations }` then we don't need the `hasOperationSupportingMultipart` helper anymore... because in the template we can just check `{{#if containsMultipartOperations}}` (same where we generate `api.hbs`)